README.md

```diff
@@ -51,7 +51,7 @@ This is the result of an initial GSoC contribution by @[ah450](https://github.co
 
 1. **PARSER** (for parsing any format):
 * Use `pyspdxtools_parser --file <filename>` where `<filename>` is the location of the file.              
-Try running: `pyspdxtools_parser --file data/SPDXRdfExample.rdf`.
+Try running: `pyspdxtools_parser --file tests/data/formats/SPDXRdfExample.rdf`.
```

Comment:
The example files in `test/data/formats/` are not included in built distributions, only in source distributions (the same is true for the example files in `data/`, for that matter). We should rephrase these comments to something like "If you are using a source distribution, try running..."